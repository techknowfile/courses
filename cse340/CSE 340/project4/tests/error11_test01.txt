TYPE
    x, y, z: INT;
    y, r : REAL;
{
    a = 1;
}