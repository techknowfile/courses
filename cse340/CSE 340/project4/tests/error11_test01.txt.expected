ERROR CODE 1.1 y
